edu-services/gradebook-service/impl/src/java/org/sakaiproject/component/gradebook/GradebookServiceHibernateImpl.java

```diff
@@ -1990,7 +1991,9 @@ private boolean isGradeValid(String grade, int gradeEntryType, LetterGradePercen
 				  }
 			  } catch (NumberFormatException nfe) {
 				  if (log.isDebugEnabled()) log.debug("Passed grade is not a numeric value");
-			  }
+			  } 	catch (ParseException pe) {
```

Comment:
This would be cleaner written as a multi-catch and you can remove the log.isDebugEnabled()

catch (NumberFormatException | ParseException e) { 
  log.debug(   );
}